### DISCUSSION TITLE
anyone build dell precision T3640 with this EFI ?

### DISCUSSION THREAD

**Discussion by FatShen3** [billzhong/dell-precision-3240-compact-hackintosh#3] [Q&A]
I got T3640 and i failed build it with this EFI. It stucks at "set packet allocation type to 1"

**Comment by FatShen3** [comment]
and if I use my own EFI, i always got this error:
"panic (cpu 0 caller)", and the "0" can also be "2"、"6" etc.

**Reply by billzhong** [reply]
Can you provide more details?
